docs/semgrep-supply-chain/overview.md

```diff
@@ -21,121 +21,112 @@ Object.entries(frontMatter).filter(
+If Semgrep Supply Chain determines that the codebase leverages a package
+containing a vulnerability, it generates a **finding**. If Semgrep Supply Chain
+also determines that there's a code pattern in the codebase that matches the
+vulnerability definition, then the finding is flagged as **reachable**
+(otherwise, it is **unreachable**).
 
-In Semgrep Cloud Platform, specific findings of a dependency and code match are called **usages**. Usages are grouped by their **vulnerability**. Vulnerabilities in Semgrep Supply Chain typically have a CVE number corresponding to the record in the [CVE Program](https://www.cve.org/About/Overview).
+If Semgrep Supply Chain detects the use of a vulnerable version of a dependency,
+but it doesn't have a reachability rule, then the finding is **undetermined**
+(this is most common with older vulnerabilities or vulnerabilities with lower
+severity levels).
 
-Semgrep Cloud Platform also includes a list of **Advisories** for reference. Advisories include all vulnerabilities covered by Semgrep Supply Chain, regardless of whether the related dependency is used in scanned code.
+Undetermined findings are generated by Semgrep Supply Chain's
+**[lockfile-only](/semgrep-supply-chain/glossary/#lockfile-only-rules)** rules,
+which are rules that do not perform reachability analysis. These findings are
+generated by checking the dependency's version listed in the lockfile or
+manifest against a list of versions with known vulnerabilities.
 
-<AdmonitionSotCves />
+Specific dependency and code match findings are called **usages**, which are
+grouped by vulnerability. For each vulnerability, SCP also displays a CVE number
+corresponding to the [CVE program record](https://www.cve.org/About/Overview).
 
-The following diagram displays the relationship between a Supply Chain rule, the lockfile, and the codebase being scanned:
+### Transitive dependencies and reachability analysis
 
-![Relationship between a Supply Chain rule, lockfile, CVE record, and codebase](/img/sc-reachability-analysis.png)
-_Figure 2_. Relationship between a Supply Chain rule, lockfile, CVE record, and codebase.
+A [transitive
+dependency](/docs/semgrep-supply-chain/glossary/#transitive-or-indirect-dependency),
+also known as an indirect dependency, is a dependency of a dependency. Semgrep
+Supply Chain scans transitive dependencies for security vulnerabilities, but it
+does *not* perform reachability analysis. This means that Semgrep Supply Chain
+doesn't check the source code of your project's dependencies to determine if
+their dependencies produce a reachable finding in your code.
 
-## Semgrep and Semgrep Supply Chain
```

Comment:
Also, why did we remove the table?

Reply:
I plan on bringing this back later when we have a high-level doc (About Semgrep) or maybe plunking it onto the homepage; if you think the chart should come back, I am not opposed

Reply:
We have a "secret" list of all features of Semgrep in the pricing-and-billing page (it's not a secret, just not very discoverable). We should keep that table somewhere in live, feel free if you want to move it out of SSC and into the pricing-and-billing doc; or if you want to modularize it and import to more than 1 place. 

Note that the bit about how our product categorizes a finding, especially the "removed" status unique to Semgrep Code in particular, is pretty important, as well as how we automatically categorize stuff as fixed or not.

Reply:
Got it -- I've updated the chart a tad and moved it to pricing and billing.